contracts/pol/CarbonPOL.sol

```diff
@@ -308,7 +308,7 @@ contract CarbonPOL is ICarbonPOL, Upgradeable, ReentrancyGuardUpgradeable, Utils
         } else {
             inputAmount = _sellTokenForETH(token, amount);
         }
-        emit TokenTraded(msg.sender, token, amount, inputAmount);
+        emit TokenTraded(msg.sender, token, inputAmount, amount);
```

Comment:
This is why I always recommend using the key-value notation:

`emit TokenTraded({caller: msg.sender, token: token, inputAmount: inputAmount, ooutputAmount: amount})`

Reply:
Changed to the key-value notation